torch/testing/_internal/common_methods_invocations.py

```diff
@@ -13183,6 +13205,32 @@ def ref_pairwise_distance(input1, input2):
            ),
            sample_inputs_func=sample_inputs_atleast1d2d3d,
            ),
+    OpInfo('flatten',
+           dtypes=all_types_and_complex_and(torch.bool, torch.float16, torch.bfloat16),
+           supports_out=False,
+           supports_forward_ad=True,
+           supports_fwgrad_bwgrad=True,
+           sample_inputs_func=sample_inputs_flatten,
+           skips=(
+               # File "pytorch/torch/testing/_internal/common_jit.py", line 306, in test_type
+               # self.assertTrue(actual_type.isSubtypeOf(out_type))
+               # AssertionError: False is not true
+               DecorateInfo(unittest.skip("Skipped!"), 'TestJit', 'test_variant_consistency_jit'),
```

Comment:
I'll update the error here but do you mind cc'ing on skipped jit failures / test failures in the future ? this is a legitimate error. admittedly, there are a number of flakey failures, which https://github.com/pytorch/pytorch/issues/69320 should help with.

Reply:
sorry it's a bit hard to figure out if these jit failures are flaky or real concerns since we already have so many skips. but point taken, will cc in future to cross-check.

Reply:
yea, fair. i'll try to prioritize #69320 and other fixes so it's less frequent..